### PR TITLE
Makes stock parts build 5x faster

### PR DESCRIPTION
**Pull request by monster860**
:cl: monster860
tweak: Makes stock parts build 5x faster
/:cl:

### CODE REVIEW

**Comment by Kevak** [comment]
_Finally!_ Thank you!

**Comment by Fox-McCloud** [comment]
Unironically buffing science.

**Comment by fludd12** [comment]
"""Buffing""" the print rate of parts that almost never get printed anyway

**Comment by Glorken** [comment]
And when you have to print them, you have to print fifteen million.

**Comment by Ssirius** [comment]
The slow part print speed is what made me stop upgrading the station when I used to do R&D, while sure it does technically buff science a bit, science is not the department that is going to suffer if the print speed remains the same, it always gets upgraded first, and sometimes its the only department that does, this would just make the rest of the station get that sweet sweet stuff more often be that a good or a bad thing.
  :+1:

**Comment by ProperPants** [comment]
Bless you :pray:

File: code/modules/research/rdconsole.dm [hunk 1]
```diff
@@ -431,6 +431,10 @@ proc/CallMaterialName(ID)
 
 					var/P = being_built.build_path //lets save these values before the spawn() just in case. Nobody likes runtimes.
 					var/O = being_built.locked
+					
+					if(ispath(P, /obj/item/weapon/stock_parts))
```
Comment (TheDZD) [inline code/modules/research/rdconsole.dm]: There has to be a better way to go about doing it than this.

Comment (monster860) [inline code/modules/research/rdconsole.dm]: What do you recommend?

Comment (PJB3005) [inline code/modules/research/rdconsole.dm]: Variable on designs as speed multiplier, then use that.

Comment (Crazylemon64) [inline code/modules/research/rdconsole.dm]: Designs already have a `construction_time` var - Make the console use that var, then set this var lower for the stock parts.

**Comment by taukausanake** [comment]
I think the current times are alright. Building single parts is quick, building bulk takes time as it should. I don't think of it as buffing Science. I think of this as being too lazy to wait

**Comment by Spacemanspark** [comment]
I don't find any particular issue with this.

**Comment by KasparoVy** [comment]
Maybe I've just gotten used to waiting. I don't feel this was an issue to begin with, let alone one worth correcting to this degree.

**Comment by TheDZD** [comment]
Are you going to be making changes to this so that the build time is reduced in a non-shitty way?

**Comment by Fox-McCloud** [comment]
@TheDZD 

Admittedly, it's going to be hard, because build times are meant to be universal on the protolathe, and construction time on all other pieces are 0, by default--in addition to this, you have some non-zero construction times on other things--largely for the mech fabricator.

It's...not ideal though, I would agree with you there.

**Comment by monster860** [comment]
@Fox-McCloud @TheDZD Done

**Pull request merged**